tools/ci_build/github/linux/test_custom_ops_pytorch_export.sh

```diff
@@ -0,0 +1,6 @@
+
+pip3 install --user numpy==1.19.0 torch pytest
+
+export PYTHONPATH="/onnxruntime_src/tools"
```

Comment:
not sure whether this is your issue or not, but usually we append content to PYTHONPATH as opposed to override it. Try

`export PYTHONPATH="/onnxruntime_src/tools:${PYTHONPATH}"`